base/linalg/lq.jl

```diff
@@ -17,10 +17,29 @@ end
 LQ{T}(factors::AbstractMatrix{T}, τ::Vector{T}) = LQ{T,typeof(factors)}(factors, τ)
 LQPackedQ{T}(factors::AbstractMatrix{T}, τ::Vector{T}) = LQPackedQ{T,typeof(factors)}(factors, τ)
 
+"""
+    lqfact!(A) -> LQ
+
+Compute the LQ factorization of `A`, using the input
```

Comment:
should we reference somewhere what this means if you've never heard of it before?